vspace/src/main/webapp/WEB-INF/views/staff/modules/sequences/sequence.jsp

```diff
@@ -1,23 +1,207 @@
+	<li class="breadcrumb-item"><a
+		href="<c:url value="/staff/module/${module.id}" />">${module.name}</a></li>
+	<li class="breadcrumb-item active">${sequence.name}</li>
 </ol>
 
-<h1>Sequence: ${sequence.name}</h1>
+
+
+<script>
+
+
+$(document).ready(function() { 
+ 
+	
+    //-------- edit contentblock description Sequence starts --------
+    $("#submitDescription").hide()
+    $("#cancelEditDescription").hide()	
+    
+    $("#editDescription").click(function() {
+    	
+    	// using data-attributes
+    	var description = $("#description").data('value')  // gets value
+        $('<textarea id="newDescription" style="margin-top: 1%;" class="form-control" type="text" data-value="">'+description+'</textarea>').insertBefore( "#description" );
+        $("#description").hide()
+        $("#newDescription").val(description)
+        $("#newDescription").data('value',description);
+        $("#editDescription").hide()
+        $("#submitDescription").show()
+        $("#cancelEditDescription").show()
+        
+        
+    });
+
+    
+    // --- Submit Description ------
+    
+    $("#submitDescription").click(function() {
+        var formData = new FormData();
+        formData.append('description', $("#newDescription").val());
+        $.ajax({
+        url: "<c:url value="/staff/module/${module.id}/sequence/${sequence.id}/edit/description?${_csrf.parameterName}=${_csrf.token}" />",
+        type: 'POST',
+        cache       : false,
+        contentType : false,
+        processData : false,
+        data: formData,
+        enctype: 'multipart/form-data',
+        success: function(data) {
+            // replace text box with new description
+            $("#submitDescription").hide()
+            $("#cancelEditDescription").hide()
+            $("#editDescription").show()
+            var val = $("#newDescription").val();
+            $("#editDescription").show()
+            $("#description").show() 
+            $('#description').insertBefore( "#newDescription" );
+            $("#description").text(val);
+            $("#description").data('value',val);
+            $("#newDescription").hide()
+        },
+        error: function(data) {
+            $(".open").removeClass("open");
+                var alert = $('<div class="alert alert-danger alert-dismissible fade show" role="alert"><p>We are sorry but something went wrong. Please try again later.</p><button type="button" class="close" data-dismiss="alert" aria-label="Close"><span aria-hidden="true">&times;</span></button></div>');
+                $('.error').append(alert);
+            }
+        });
+        
+   });
+    
+    // ---- Cancel Edited Description ----
+    
+    $("#cancelEditDescription").click(function(){
+        $("#submitDescription").hide()
+        $("#editDescription").show()
+        $("#cancelEditDescription").hide()
+        var description = $("#description").data('value')
+        $('description').insertBefore( "#newDescription" );
```

Comment:
see above

Reply:
see above

Reply:
see above (also, this doesn't refer to anything 'description' would refer to a tag, '#description' refers to an element with id description, so if this works, you can delete this line).

Reply:
same here